src/replacements/list/$elm$core$List$filterMap.js

```diff
@@ -0,0 +1,13 @@
+var $elm$core$List$filterMap = F2(function (f, xs) {
+  var tmp = _List_Cons(undefined, _List_Nil);
+  var end = tmp;
+  for (; xs.b; xs = xs.b) {
+    var m = f(xs.a);
+    if (!m.$) {
```

Comment:
Should this actually check for `m.a`? I'm assuming that every elm type has a `$` 🤔

Reply:
The check is not about the presence of the $ field, it's about whether the Maybe returned by the function is Nothing ($ = 1 IIRC) or Just ($ = 0).